app/components/RandomQuote/RandomQuote.js

```diff
@@ -35,6 +35,15 @@ const RandomQuote = (props: Props) => {
     currentQuote
   } = props;
 
+  const [seenQuotes, setSeenQuotes] = useState([]);
+
+  useEffect(() => {
+    const quoteId = props.currentQuote.id;
+    if (!seenQuotes.includes(quoteId)) {
+      setSeenQuotes([...seenQuotes, quoteId]);
```

Comment:
Since this will result in a re-render (and we need no re-render), using `useRef` may be more suitable. 

https://stackoverflow.com/questions/56455887/react-usestate-or-useref